io.github.materialgram.yml

```diff
@@ -0,0 +1,553 @@
+app-id: io.github.materialgram
```

Comment:
this app-id in invalid.

https://docs.flatpak.org/en/latest/conventions.html#application-ids